.github/workflows/go-lint.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         with:
           # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-          version: v1.56.0
+          version: v1.57.2
```

Comment:
The reason for bumping the version here is to resolve an unexpected lint issue in CI:
https://github.com/rilldata/rill/actions/runs/8511596615

Removed `timeCmpSimplify` also because the latest release has dropped this function.